lib/plurimath/mathml/parser.rb

```diff
@@ -37,6 +37,12 @@ def parse_nodes(nodes)
           end
         end
       end
+
+      def validate_attributes(attributes)
+        attrs = %w[columnlines mathvariant mathcolor notation close open].freeze
```

Comment:
If these are always constant, define them outside of the method as class constants.